client/extensions/woocommerce/app/promotions/promotion-update.js

```diff
@@ -244,7 +244,7 @@ function mapStateToProps( state, ownProps ) {
 	const site = getSelectedSiteWithFallback( state );
 	const currencySettings = getPaymentCurrencySettings( state );
 	const currency = currencySettings ? currencySettings.value : null;
-	const promotionId = ownProps.params.promotion;
+	const promotionId = ownProps.params.promotion_id;
```

Comment:
This sticks out a bit because it's the only ID not parsed as `Number()`.

Apparently, promotions selectors also accept it as an object (I assume as `promotion` object), but in this particular case I'd say it's definitely an ID.
Question is: it's a numeric ID (and then, let's parse it as such!), or a string?